apps/website/src/server/trpc/router/admin/show-and-tell.ts

```diff
@@ -29,6 +33,54 @@ export const adminShowAndTellRouter = router({
     .input(z.string().cuid())
     .query(({ input }) => getPostById(input)),
 
+  getModComments: permittedProcedure
+    .input(z.string().cuid())
+    .query(async ({ input }) => {
+      const comments = await getModCommentsForEntry(input);
+      if (!comments) {
+        throw new TRPCError({
+          code: "NOT_FOUND",
+          message: "Comments not found",
+        });
+      }
+      return comments;
```

Comment:
Is the explicit 404 needed or can this just return the empty result?
```suggestion
      return getModCommentsForEntry(input);
```